spacy/pipeline/spancat.py

```diff
@@ -373,28 +397,6 @@ def initialize(
         else:
             self.model.initialize()
 
-    def score(self, examples: Iterable[Example], **kwargs) -> Dict[str, Any]:
-        """Score a batch of examples.
-
-        examples (Iterable[Example]): The examples to score.
-        RETURNS (Dict[str, Any]): The scores, produced by Scorer.score_cats.
-
-        DOCS: https://spacy.io/api/spancategorizer#score
-        """
-        validate_examples(examples, "SpanCategorizer.score")
-        self._validate_categories(examples)
-        kwargs = dict(kwargs)
-        attr_prefix = "spans_"
-        kwargs.setdefault("attr", f"{attr_prefix}{self.key}")
-        kwargs.setdefault("labels", self.labels)
```

Comment:
Odd that this was in the codebase, `score_spans` doesn't actually use a `labels` argument...

Reply:
Yeah, I think there were a number of copy-paste problems with the scoring, but nothing major other than `multi_label`.